obvious-ci.docker/linux64_obvci/Dockerfile

```diff
@@ -2,34 +2,51 @@ FROM centos:6
 
 MAINTAINER Phil Elson <pelson.pub@gmail.com>
 
-RUN yum install -y \
+
+# Set an encoding to make things work smoothly.
+ENV LANG en_US.UTF-8
+
+# Install basic requirements.
+RUN yum update -y && \
+    yum install -y \
                    bzip2 \
                    gcc-c++ \
                    git \
                    make \
                    patch \
                    tar \
-                   wget \
                    which \
                    libXext \
                    libXrender \
                    libSM \
-                   libX11-devel
+                   libX11-devel && \
+    yum clean all
 
-RUN wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh --no-verbose && \
+# Download and install tini for zombie reaping.
+RUN curl -s -L -O https://github.com/krallin/tini/releases/download/v0.9.0/tini && \
```

Comment:
Good point. We probably could do this.

`ADD` is a very odd command. It can do some unexpected things (like decompressing some tarballs).

While I don't know exactly how `ADD` performs its magic, I do know `RUN` won't work if there is no `/bin/sh` in the image (see [comment]( https://github.com/docker/docker/issues/17896#issuecomment-155829064 )). I have tried to avoid these fancy features of `ADD` as it remains unclear to me when they will or will not work.